File: WebApplication3/WebApplication3/Models/Category.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication3.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Category")]
    public partial class Category
    {
        [Key]
        public int Id_Category { get; set; }

        public string NameCategory { get; set; }

        public string PictureCategory { get; set; }

        public string JoinDate { get; set; }

        public string Note { get; set; }

    }
}

File: Project_DoAnNhanhSOL/app/src/main/java/com/example/project_doannhanhsol/DTO/Category.java
package com.example.project_doannhanhsol.DTO;

public class Category {
    private int Id_Category;
    private  String NameCategory,PictureCategory,JoinDate,Note;

    public Category(int id_Category, String nameCategory, String pictureCategory, String joinDate, String note) {
        Id_Category = id_Category;
        NameCategory = nameCategory;
        PictureCategory = pictureCategory;
        JoinDate = joinDate;
        Note = note;
    }

    public int getId_Category() {
        return Id_Category;
    }

    public String getNameCategory() {
        return NameCategory;
    }

    public String getPictureCategory() {
        return PictureCategory;
    }

    public String getJoinDate() {
        return JoinDate;
    }

    public String getNote() {
        return Note;
    }
}
